dockers/docker-ptf-saithrift/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker-ptf
+
+RUN apt-get update
+
+COPY ["deps/saithrift-0.9.tar.gz", "/deps/"]
+COPY ["*.ini", "/etc/ptf/"]
+COPY ["deps/SAI/test/saithrift/tests/*", "/usr/share/ptf-tests/"]
+
+RUN tar xzf /deps/saithrift-0.9.tar.gz -C /usr/share/; \
+    cd usr/share/saithrift-0.9;   \
+    python setup.py install;      \
+    rm -rf /deps
```

Comment:
I think we should keep them in an independent docker though the dockerfile itself is not platform-specific. As else we will introduce a dependency of saithrift package into docker-ptf.